src/app/modules/audience/views/engagement/syndication/syndication.component.ts

```diff
@@ -259,7 +259,8 @@ export class SyndicationComponent extends EngagementBaseReportComponent {
 
   public _openLink(data): void {
     if (data && data.referrer) {
-      window.open(data.referrer, '_blank');
+      const link = data.referrer.startsWith('http') ? data.referrer : `https://${data.referrer}`;
```

Comment:
@diamond-darrell - why force https if no protocol is defined?

Reply:
you have to provide protocol otherwise a link will be opened under current url (current_url/data.referrer)

Reply:
@diamond-darrell - I understand this, my question is why force https and not use http? Probably most links without protocol will use http and not https.